website/docs/reference/datasources/querying-elasticsearch.md

```diff
@@ -1,93 +1,154 @@
 ---
 sidebar_position: 5
+description: Connect Appsmith to your Elasticsearch data and create queries.
 ---
 
 # Elasticsearch
 
-:::note
-The following document assumes that you understand the [basics of connecting to databases on Appsmith](/core-concepts/connecting-to-data-sources/connecting-to-databases.md#connecting-to-a-database). If not, please go over them before reading further.
-:::
+This page describes how to connect to your Elasticsearch database and query it from your Appsmith app.
+
+## Connection parameters
+
+The following is a reference guide that provides a description of the parameters for connecting to Elasticsearch.
 
-## Connection settings
+<figure>
+  <img src="/img/elasticsearch-datasource-config.png" style= {{width:"100%", height:"auto"}} alt="Connect to Elasticsearch"/>
+  <figcaption align = "center"><i>Connect to Elasticsearch</i></figcaption>
+</figure>
 
-The Elasticsearch plugin requires two pieces of information to establish a connection.
+<dl>
+  <dt><b>Host URL</b></dt>
+  <dd>The network location where your Elasticsearch data is hosted. This can be a domain name or an IP address. To connect to a local database, see <a href="/advanced-concepts/more/how-to-work-with-local-apis-on-appsmith"><b>Connect Local Database</b></a> for directions. </dd><br />
 
-![Click to expand](/img/elasticsearch-datasource-form.png)
+  <dt><b>Port</b></dt>
+  <dd>The port number to connect to on the server. </dd><br />
 
-* **Host Address / Port\*:** Fill in the elastic search instance's address and port. This field supports multiple endpoints if that be the need.
-* **Username / Password:** The authentication detail for the elastic search instance. The password for your data source is encrypted when it is stored within the Appsmith database.
-* **Authorization Header:** In case you choose to connect to your instance with another mechanism, you can use the `Authorization Header` field. This field is only considered when the `Username` and `Password` fields are empty.
+  <dt><b>Username/Password for Basic Auth</b></dt>
+  <dd>The account credentials used to log in to Elasticsearch.</dd><br />
 
-After filling up the fields as described, click the "Test" button to verify the configuration and click **Save**.
+  <dt><b>Authorization Header</b></dt>
+  <dd>Instead of the username/password fields, you can provide an <b>Authorization Header</b> to authenticate your queries. This field is only used when the <b>Username/Password for Basic Auth</b> fields are empty.</dd><br />
+</dl>
 
-## Querying Elasticsearch
+## Query Elasticsearch
```

Comment:
This section still looks wrong. Apologies if there was any confusion. 

The query configuration parameters only need to be described once. Under each parameter, you can give any number of relevant examples. 

For eg: 

- Method - Description
   - _Options_
      - GET - Description
      - POST - Description
      - PUT - Description
      - DELETE - Description
- Path - Description
  Example: `/movies/_doc/2`  
- Body - Description
   Example - here give examples of search documents, create documents, etc.